config/staging.json

```diff
@@ -1,5 +1,8 @@
 {
   "project": "terra-bard-staging",
   "orchestrationRoot": "https://firecloud-orchestration.dsde-staging.broadinstitute.org",
-  "samRoot": "https://sam.dsde-staging.broadinstitute.org"
+  "samRoot": "https://sam.dsde-staging.broadinstitute.org",
+  "bigQueryProject": "broad-dsde-prod-analytics-dev",
+  "bigQueryDatasetId": "warehouse",
```

Comment:
I think this should be the scratch one to start.  Mind confirming with Alex?

Reply:
This _might_ be moot if we go the logging approach.

Reply:
Ah so we still need to define which bigQuery dataset to send the logs to